src/api/masJobRecord/GetCreatorMasJobRecords.ts

```diff
@@ -0,0 +1,48 @@
+/* Import Dependencies */
+import axios from 'axios';
+import KeycloakService from 'app/Keycloak';
+
+/* Import Types */
+import { MasJobRecord, JSONResultArray } from 'app/Types';
+
+/* Import Exceptions */
+import { NotFoundException } from 'app/Exceptions';
+
+
+/**
+ * Function for fetching a the logged in user's MAS job records
+ * @returns Array of MAS job records
+ */
+const GetCreatorMasJobRecords = async () => {
+    let creatorMasJobRecords: MasJobRecord[] = [];
+
+    const token = KeycloakService.GetToken();
+
+    if (token) {
+        try {
+            const result = await axios({
+                method: 'get',
+                url: `mjr/creator/${KeycloakService.GetSubject()}`,
```

Comment:
No it gets the keycloak identifier, copied as it was in the old version.
Should it be orcid?
I would have assumed I only have to pass the bearer and the back-end does the check for orcid, like the annotation/creator endpoint, is that possible?

Reply:
the endpoint `mjr/creator/{creator ID}` currently needs the orcid as the creator id. 
it might make more sense to get the creator ID directly from the auth token, actually, that way you can't see jobs other users have scheduled. will bring this up in the sprint planning.

Reply:
for now use the orcid, but be aware this may change in the next sprint

Reply:
Changed it to ORCID